edx/edx-django-utils
# Moving logging filters from edx-platform

**Description:**

Moving logging filters for user and remote
IP to this repo, so they can be re-used by
newly created IDAs through the cookie-cutter
template.

**JIRA:**

https://openedx.atlassian.net/browse/SEG-34

**Dependencies:**

None

**Merge deadline:**

None

**Installation instructions:**

None

**Testing instructions:**

1. Update edx-django-utils to latest version
2. Confirm that the UserFilter and RemoteIpFilter can
be imported from edx_django_utils.logging.filters

**Reviewers:**

**Merge checklist:**

**Post merge:**
      finished.

**Author concerns:**

None
